src/test/fuzz/cluster_linearize.cpp

```diff
@@ -241,103 +241,149 @@ std::vector<ClusterIndex> ReadLinearization(const DepGraph<BS>& depgraph, SpanRe
 
 } // namespace
 
```

Comment:
`clusterlin: merge several DepGraph fuzz tests into simulation test`

Seems to only merge 2 fwiw, might just enumerate them?